proposals/2020-11-stargate-upgrade/README.md

```diff
@@ -0,0 +1,133 @@
+
+We also completed and delivered a completed legacy Amino Audit, [AminoRest & You](https://github.com/cosmosdevs/stargate/blob/master/audit.md). The core finding of this audit was that changes to the underlying structs result in an interface that is close to the prior version allowing legacy queries to return valid data.
+
+### 3. Documentation and Self-Certification
+
+Documentation was a success for the Cosmos Stargate effort. The upgrades with the most breaking changes such as legacy Amino have a complete audit with documentation on exceptions for blockchain API interfaces.
+
+Most exchanges and wallets have taken self-certification on directly. Our team continues to provide real-time support on multiple partner slack channels and on the Cosmos [#stargate Discord](https://discord.gg/W8trcGV) channel.
+
+### Cosmos Stargate Integration Success
+
+Cosmos Stargate integration success with exchanges and wallet providers reflects the quality of the code developed by the various Cosmos teams.
+
+## Prop 29
+
+This upgrade also implements the fund recovery procedure defined in proposal 29. The code is [here](https://github.com/cosmos/gaia/blob/main/app/prop29.go). As requested during the discussion of the governance proposal, the migration and fund recovery procedure verifies all signatures. This code was tested during the cosmoshub-3 simulated upgrade testnet.ƒ
+
+## Stargate Upgrade Steps
+
+------------
+The upgrade steps for the simulated upgrade of the current Cosmoshub mainnet to Cosmos Stargate includes the following steps:
+
+This section is with the current `gaia 2.0.*` implementation.
+
+  1. Validators should set their `gaia` with a halt time of `1611813600`ie. 06:00 UTC on Jan 28th in Unix time.
+
+  2. Validators should then export the current cosmos state with `gaiad export > cosmoshub-3-export.json`
+
+  3. Validators should determine the height of the last block as `last-cosmoshub-3_block`.
+
+  4. Validators should back up their `.gaiad` directory.
+
+This section is with the upgrade `gaia 3.0.*` implemenataion.
+
+  1. Validators should then migrate the exported genesis state. `gaiad migrate cosmoshub-3-export.json --chain-id=cosmoshub-4 --initial-height [last_cosmoshub-3_block+1] > genesis.json`
+  2. Validators should delete their `~/.gaiad` directory and create a new one with `gaiad init [name]` and then edit their configuration files.
+  3. Validators should then start `cosmoshub-4` with `gaiad start`. Automated audits of the genesis state can take 30-120 min using the crisis module. This can be disabled by `gaiad start --x-crisis-skip-assert-invariants`.
+
+Validators should expect that at least 16GB of RAM needs to be provisioned to process block 1 on cosmoshub-4.
+
+## Time of the Upgrade
+
+------------
+We propose scheduling the Cosmoshub-3 to Cosmoshub-4 simulated upgrade for Thursday Jan 28th, 2021 at 0600 UTC
```

Comment:
Should the Gaia/Cosmos SDK/Tendermint Core engineering teams be on standby at this time?

Reply:
probably not all of them but I think a few ppl would need to monitor the upgrade